cpu/atmega_common/thread_arch.c

```diff
@@ -34,7 +34,7 @@
  * local function declarations  (prefixed with __)
  */
 static void __context_save(void);
-static void __context_restore(void);
+static void atmega_context_restore(void);
```

Comment:
Because of consistency, we can't leave one function with namespace `atmega_` and not the others.

Reply:
I somehow overlooked the others. Thanks for spotting this!